src/OpenLoco/src/Windows/CompanyList.cpp

```diff
@@ -27,34 +27,43 @@ using namespace OpenLoco::Literals;
-    static loco_global<uint16_t, 0x0113DC84> _graphXOffset;
-    static loco_global<uint32_t, 0x0113DC86> _graphYAxisLabelIncrement;
-    static loco_global<uint16_t, 0x0113DC8A> _graphLineCount;
-    static loco_global<uint32_t[32], 0x0113DC8C> _graphYData;
-    static loco_global<uint32_t, 0x0113DD0C> _graphDataTypeSize;
-    static loco_global<uint16_t[32], 0x0113DD10> _graphDataStart;
-    static loco_global<uint32_t, 0x0113DD50> _dword_113DD50;
-    static loco_global<PaletteIndex_t[32], 0x0113DD54> _graphLineColour;
-    static loco_global<uint16_t, 0x0113DD74> _graphDataEnd;
-    static loco_global<uint16_t, 0x0113DD76> _graphXLabel;
-    static loco_global<uint32_t, 0x0113DD78> _graphXAxisRange;
-    static loco_global<uint32_t, 0x0113DD7C> _dword_113DD7C;
-    static loco_global<uint16_t, 0x0113DD80> _word_113DD80; // graphXAxisIncrement?
-    static loco_global<uint16_t, 0x0113DD82> _graphXAxisLabelIncrement;
-    static loco_global<uint16_t, 0x0113DD84> _graphYLabel;
-    static loco_global<uint32_t, 0x0113DD86> _dword_113DD86;
-    static loco_global<uint32_t, 0x0113DD8A> _dword_113DD8A;
-    static loco_global<uint32_t, 0x0113DD8E> _dword_113DD8E;
-    static loco_global<uint8_t, 0x0113DD99> _byte_113DD99;
-    static loco_global<uint16_t[32], 0x0113DD9A> _graphItemId;
+
+#pragma pack(push, 1)
+    struct GraphSettings
+    {
+        uint16_t left;                 // 0x0113DC7A
+        uint16_t top;                  // 0x0113DC7C
+        uint16_t right;                // 0x0113DC7E
+        uint16_t bottom;               // 0x0113DC80
+        uint16_t yOffset;              // 0x0113DC82
+        uint16_t xOffset;              // 0x0113DC84
+        uint32_t yAxisLabelIncrement;  // 0x0113DC86
+        uint16_t lineCount;            // 0x0113DC8A
+        uint32_t* yData[32];           // 0x0113DC8C
+        uint32_t dataTypeSize;         // 0x0113DD0C
+        uint16_t dataStart[32];        // 0x0113DD10
+        uint32_t dword_113DD50;        // 0x0113DD50
+        PaletteIndex_t lineColour[32]; // 0x0113DD54
+        uint16_t dataEnd;              // 0x0113DD74
+        uint16_t xLabel;               // 0x0113DD76
+        uint32_t xAxisRange;           // 0x0113DD78
+        uint32_t dword_113DD7C;        // 0x0113DD7C
+        uint16_t word_113DD80;         // 0x0113DD80 -- graphXAxisIncrement?
+        uint16_t xAxisLabelIncrement;  // 0x0113DD82
+        uint16_t yLabel;               // 0x0113DD84
+        uint32_t dword_113DD86;        // 0x0113DD86
+        uint32_t dword_113DD8A;        // 0x0113DD8A
+        uint32_t dword_113DD8E;        // 0x0113DD8E
+        uint8_t pad_113DD92[7];        // 0x0113DD92
+        uint8_t byte_113DD99;          // 0x0113DD99
+        uint16_t itemId[32];           // 0x0113DD9A
+    };
+#pragma pack(pop)
+
+    static_assert(sizeof(GraphSettings) == 0x0113DD9A + 64 - 0x0113DC7A);
+    static loco_global<GraphSettings, 0x0113DC7A> _graphSettings;
```

Comment:
Is this still required to be interop?

Reply:
Yes, the actual graph plotting in `drawGraph` hasn't been implemented yet! I'd like the code for it to live in another file, though. This kind of struct should allow us to  bridge that gap, and to easily add graphs to other areas in the future.

Reply:
Oh right.